src/app/components/social-system/add/add.component.ts

```diff
@@ -35,16 +38,33 @@ export class AddComponent implements OnInit {
 
       next: data =>
       { 
-          console.log("Request wurde erfolgreich gesendet")
+          console.log("Request wurde erfolgreich gesendet");
+          this.sentRequests.push(userId);
       },
       error: (error) =>
 
       console.log(error.message)
     });
 
+   var btn = document.getElementById("requestButton"+userId);
+   btn!.innerHTML = "Pending";
+   btn?.setAttribute("disabled","disabled");
```

Comment:
I'm quite unsure on how to do this since the problem is that every request has it's own button  which is being identified by the id of the request and I wouldn't know how to access it, since I'v already tried @ViewChild which was not possible